components/dataRepresentation/index.less

```diff
@@ -74,26 +73,31 @@
     }
 
     .noDataInfo {
-        @loadingIndicatorSize: 20px;
-        border-bottom: 1px solid @borderColor;
-        border-top: 1px solid @borderColor;
-        margin-top: -1px;
-        padding: 10px 14px;
-        min-height: 2 * 10px + @loadingIndicatorSize;
-        font-size: 14px;
-        color: @darkGrey;
-
-        display: table;
-        width: 100%;
+        text-align: center;
+        padding: 0 40px;
+
+        .head {
+
+        }
 
-        > * {
-            display: table-cell;
-            vertical-align: middle;
+        .glyphs {
+            width: 60%;
+            max-width: 160px;
+            margin: auto;
+            display: flex;
+
+            img {
+                width: 100%;
+                height: 100%;
+                display: block;
+                padding: 5px;
+            }
         }
 
-        > #loadingIndicator {
-            width: @loadingIndicatorSize; height: @loadingIndicatorSize;
-            float: right;
+        .sub {
+            opacity: 0.75;
```

Comment:
colour here is `#BCC2D0` in the design, although i think `@mediumGrey` would work